compute_coverage_target.R

```diff
@@ -65,7 +65,23 @@ if(is.labeled){
     stopifnot(is.character(penalty.str))
     pre <- paste0(prob.cov.bedGraph, "_penalty=", penalty.str)
     penalty_segments.bed <- paste0(pre, "_segments.bed")
-    if(!file.exists(penalty_segments.bed)){
+    penalty_loss.tsv <- paste0(pre, "_loss.tsv")
+    already.computed <- tryCatch({
+      penalty.segs <- fread(penalty_segments.bed, colClasses=list(NULL=5))
```

Comment:
TODO add a test case where the segments file does not have the right number of lines, and the loss file is empty